tensilelite/Tensile/Common.py

```diff
@@ -1764,6 +1765,9 @@ def assignGlobalParameters( config ):
       if 'HIP version' in line:
         globalParameters['HipClangVersion'] = line.split()[2]
         print1("# Found  hipcc version " + globalParameters['HipClangVersion'])
+      if 'AMD clang version' in line:
+        globalParameters['AMDClangVersion'] = line.split()[3]
+        print1("# Found  clang version " + globalParameters['AMDClangVersion'])
```

Comment:
it is better to use clang tool directly to check the version.
If user try to use upstream clang, hipcc won't be there.